main.cpp

```diff
@@ -359,6 +362,16 @@ int main(int argc, char ** argv) {
                 last_n_tokens.push_back(id);
             }
 
+            // replace end of text token with newline token when in interactive mode
+            if (id == llama_token_eos() && params.interactive) {
+                id = llama_token_newline.front();
+                if (params.antiprompt.size() != 0) {
+                    // tokenize and inject first reverse prompt
+                    first_antiprompt = ::llama_tokenize(ctx, params.antiprompt.front(), false);
```

Comment:
```suggestion
                    const auto first_antiprompt = ::llama_tokenize(ctx, params.antiprompt.front(), false);
```

Reply:
Ah, I missed that. Sorry! It was correct on my local branch but I messed up here.